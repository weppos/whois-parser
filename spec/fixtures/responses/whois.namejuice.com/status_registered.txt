
Domain name: NAMEJUICE.COM
Registry Domain ID: 104784662_DOMAIN_COM-VRSN
Registrar WHOIS Server: whois.namejuice.com
Registrar URL: http://www.namejuice.com
Updated Date: 2018-05-03T09:52:37Z
Creation Date: 2004-03-25T16:06:56Z
Registrar Registration Expiration Date: 2020-10-09T04:00:00Z
Registrar: BRANDON GRAY INTERNET SERVICES, INC. DBA NAMEJUICE.COM
Registrar IANA ID: 636
Registrar Abuse Contact Email: abuse@namejuice.com
Registrar Abuse Contact Phone: +1.9054152681
Domain Status: clientDeleteProhibited https://www.icann.org/epp#clientDeleteProhibited
Domain Status: clientTransferProhibited https://www.icann.org/epp#clientTransferProhibited
Registry Registrant ID: 
Registrant Name: Domain Registrar
Registrant Organization: Brandon Gray Internet Services Inc.
Registrant Street: 7100 Warden Ave
Registrant Street: Unit 8
Registrant City: Markham
Registrant State/Province: ON
Registrant Postal Code: L3R 8B5
Registrant Country: CA
Registrant Phone: +1.9054152681
Registrant Phone Ext:
Registrant Fax: +1.9054152682
Registrant Fax Ext:
Registrant Email: registrar@namejuice.com
Registry Admin ID: 
Admin Name: Domain Registrar
Admin Organization: Brandon Gray Internet Services Inc.
Admin Street: 7100 Warden Ave
Admin Street: Unit 8
Admin City: Markham
Admin State/Province: ON
Admin Postal Code: L3R 8B5
Admin Country: CA
Admin Phone: +1.9054152681
Admin Phone Ext:
Admin Fax: +1.9054152682
Admin Fax Ext:
Admin Email: registrar@namejuice.com
Registry Tech ID: 
Tech Name: Domain Registrar
Tech Organization: Brandon Gray Internet Services Inc.
Tech Street: 7100 Warden Ave
Tech Street: Unit 8
Tech City: Markham
Tech State/Province: ON
Tech Postal Code: L3R 8B5
Tech Country: CA
Tech Phone: +1.9054152681
Tech Phone Ext:
Tech Fax: +1.9054152682
Tech Fax Ext:
Tech Email: registrar@namejuice.com
Name Server: DNS1.NAMERESOLVERS.COM
Name Server: DNS2.NAMERESOLVERS.COM
Name Server: DNS3.NAMERESOLVERS.COM
Name Server: DNS4.NAMERESOLVERS.COM
DNSSEC: Unsigned
URL of the ICANN WHOIS Data Problem Reporting System: http://wdprs.internic.net/
Last update of WHOIS database: 2018-05-03T09:52:37Z

For more information on Whois status codes, please visit https://www.icann.org/resources/pages/epp-status-codes-2014-06-16-en.



The data in this whois database is provided to you for information
purposes only, that is, to assist you in obtaining information about or
related to a domain name registration record. We make this information
available "as is," and do not guarantee its accuracy. By submitting a
whois query, you agree that you will use this data only for lawful
purposes and that, under no circumstances will you use this data to: (1)
enable high volume, automated, electronic processes that stress or load
this whois database system providing you this information; or (2) allow,
enable, or otherwise support the transmission of mass unsolicited,
commercial advertising or solicitations via direct mail, electronic
mail, or by telephone. The compilation, repackaging, dissemination or
other use of this data is expressly prohibited without prior written
consent from us.

We reserve the right to modify these terms at any time. By submitting
this query, you agree to abide by these terms!
