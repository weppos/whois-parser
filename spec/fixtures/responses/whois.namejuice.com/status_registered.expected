#domain
  %s == "domain.com"

#domain_id
  %s == "104784662_DOMAIN_COM-VRSN"

#status
  %s == :registered

#available?
  %s == false

#registered?
  %s == true

#created_on
  %s %CLASS{time}
  %s %TIME{2004-03-25 16:06:56 UTC}

#updated_on
  %s %CLASS{time}
  %s %TIME{2018-05-03 09:52:37 UTC}

#expires_on
  %s %CLASS{time}
  %s %TIME{2020-10-09 04:00:00 UTC}

#registrar
  %s %CLASS{registrar}
  %s.id           == "636"
  %s.name         == "BRANDON GRAY INTERNET SERVICES, INC. DBA NAMEJUICE.COM"
  %s.organization == "BRANDON GRAY INTERNET SERVICES, INC. DBA NAMEJUICE.COM"
  %s.url          == "http://www.namejuice.com"

#registrant_contacts
  %s %CLASS{array}
  %s %SIZE{1}
  %s[0] %CLASS{contact}
  %s[0].type          == Whois::Parser::Contact::TYPE_REGISTRANT
  %s[0].name          == "Domain Registrar"
  %s[0].organization  == "Brandon Gray Internet Services Inc."
  %s[0].address       == "7100 Warden Ave, Unit 8"
  %s[0].city          == "Markham"
  %s[0].zip           == "L3R 8B5"
  %s[0].state         == "ON"
  %s[0].country_code  == "CA"
  %s[0].phone         == "+1.9054152681"
  %s[0].fax           == "+1.9054152682"
  %s[0].email         == "registrar@namejuice.com"

#admin_contacts
  %s %CLASS{array}
  %s %SIZE{1}
  %s[0] %CLASS{contact}
  %s[0].type          == Whois::Parser::Contact::TYPE_ADMINISTRATIVE
  %s[0].name          == "Domain Registrar"
  %s[0].organization  == "Brandon Gray Internet Services Inc."
  %s[0].address       == "7100 Warden Ave, Unit 8"
  %s[0].city          == "Markham"
  %s[0].zip           == "L3R 8B5"
  %s[0].state         == "ON"
  %s[0].country_code  == "CA"
  %s[0].phone         == "+1.9054152681"
  %s[0].fax           == "+1.9054152682"
  %s[0].email         == "registrar@namejuice.com"

#technical_contacts
  %s %CLASS{array}
  %s %SIZE{1}
  %s[0] %CLASS{contact}
  %s[0].type          == Whois::Parser::Contact::TYPE_TECHNICAL
  %s[0].name          == "Domain Registrar"
  %s[0].organization  == "Brandon Gray Internet Services Inc."
  %s[0].address       == "7100 Warden Ave, Unit 8"
  %s[0].city          == "Markham"
  %s[0].zip           == "L3R 8B5"
  %s[0].state         == "ON"
  %s[0].country_code  == "CA"
  %s[0].phone         == "+1.9054152681"
  %s[0].fax           == "+1.9054152682"
  %s[0].email         == "registrar@namejuice.com"

#nameservers
  %s %CLASS{array}
  %s %SIZE{4}
  %s[0] %CLASS{nameserver}
  %s[0].name == "dns1.nameresolvers.com"
  %s[1] %CLASS{nameserver}
  %s[1].name == "dns2.nameresolvers.com"
  %s[2] %CLASS{nameserver}
  %s[2].name == "dns3.nameresolvers.com"
  %s[3] %CLASS{nameserver}
  %s[3].name == "dns4.nameresolvers.com"
